src/core/annotation.js

```diff
@@ -493,10 +493,7 @@ class Annotation {
    * @private
    */
   _isViewable(flags) {
-    return (
-      !this._hasFlag(flags, AnnotationFlag.INVISIBLE) &&
-      !this._hasFlag(flags, AnnotationFlag.NOVIEW)
-    );
```

Comment:
Historically we've had bugs related to Annotation-visiblity and various flags.

Note that this code applies *to all* Annotations, and the code below (i.e. the one setting e.g. `data.display`) *only* applies to WidgetAnnotations, hence please explain why these changes are generally correct/safe!

Reply:
Oh yes you're right: I was bit FieldAnnotation-focused...